util/test/sub_test.py

```diff
@@ -848,7 +854,7 @@ def filter_errors(output_in, pre_exec_output, execgoodfile, execlog):
 #sys.stdout.write('lm=%s\n'%(chpllm))
 
 # CHPL_TASKS
-chpltasks=os.getenv('CHPL_TASKS', 'none').strip()
+chpltasks=chpl_tasks.get()
```

Comment:
I can't remember why anymore (probably a historical artifact at this point), but today start_test is what gets all the values and it then sets the env vars for sub_test to read. See for example what we do with `CHPL_COMM`. I can't say that it's a very good solution, but I might just stick with that style for consistency.